src/baseCommand.ts

```diff
@@ -48,6 +49,15 @@ export abstract class BaseCommand extends Command {
         /**
          * Configuration
          */
+
+        // This happen when providing network through env var
+        if (!UTILS.getNetworksList().includes(flags.network)) {
```

Comment:
This check is not provided by oclif? If UNS_NETWORK value is not included in options list, it should directly throw an exception  without calling Command.run()

Reply:
I know, but apparently the check is not done when the flag is set by environment variable. I guess its an oclif issue

Reply:
PR has been pushed on Oclif project https://github.com/oclif/parser/pull/64

Reply:
PR has been pushed on Oclif project oclif/parser#64